### ISSUE TITLE
Dynamic recipes

### ISSUE THREAD

**Issue by solumath** [Lolcroc/ProjectLepton#10]
Do you have in plan to implement support for dying shulker boxes?

**Comment by mont-40** [comment]
Yes there are plans to make this in a future version (along with fireworks, book/map duplication, etc.).

Duplicate of #8 and #12.
